### PR TITLE
ObjectData

### PR DESCRIPTION
**Pull request by Sergeanur**

### CODE REVIEW

File: src/objects/ObjectData.cpp [hunk 1]
```diff
@@ -18,11 +20,55 @@ CObjectData::Initialise(const char *filename)
...
+	ms_aObjectInfo[0].m_fTurnMass = 99999.0f;
+	ms_aObjectInfo[0].m_fAirResistance = 0.99f;
+	ms_aObjectInfo[0].m_fElasticity = 0.1f;
+	ms_aObjectInfo[0].m_fBuoyancy = GRAVITY * ms_aObjectInfo[0].m_fMass * 2.0f;
+	ms_aObjectInfo[0].m_fUprootLimit = 0.0f;
+	ms_aObjectInfo[0].m_fCollisionDamageMultiplier = 1.0f;
+	ms_aObjectInfo[0].m_nCollisionDamageEffect = 0;
+	ms_aObjectInfo[0].m_nSpecialCollisionResponseCases = 0;
+	ms_aObjectInfo[0].m_bCameraToAvoidThisObject = false;
+
```
Comment (Bigbossbro08) [inline src/objects/ObjectData.cpp:34]: I think it's better if it's wrapped with a function/constructor

Comment (Sergeanur) [reply src/objects/ObjectData.cpp:34]: Don't think original did that.

Comment (Bigbossbro08) [reply src/objects/ObjectData.cpp:34]: Point but I guess it can make code look little better? or was it inlined?

**Review by aap — APPROVED** [review]

File: src/objects/ObjectData.cpp (hunk 1 above)
Comment (Sergeanur) [reply src/objects/ObjectData.cpp:34]: I don't think it's inlined. We don't stray too much unless there a reason.

**Review by erorcun — APPROVED** [review]

**Review by ShFil119 — APPROVED** [review]

**Pull request merged**